docs/nodes/aws.md

```diff
@@ -272,17 +420,25 @@ Node classes list of aws provider.
 - **diagrams.aws.mobile.APIGateway**
 - **diagrams.aws.mobile.Appsync**
 - **diagrams.aws.mobile.DeviceFarm**
+- **diagrams.aws.mobile.Mobile**
 - **diagrams.aws.mobile.Pinpoint**
 
 ## aws.network
 
+- **diagrams.aws.network.APIGatewayEndpoint**
 - **diagrams.aws.network.APIGateway**
 - **diagrams.aws.network.AppMesh**
 - **diagrams.aws.network.ClientVpn**
 - **diagrams.aws.network.CloudMap**
+- **diagrams.aws.network.CloudFrontDownloadDistribution**
+- **diagrams.aws.network.CloudFrontEdgeLocation**
+- **diagrams.aws.network.CloudFrontStreamingDistribution**
 - **diagrams.aws.network.CloudFront**, **CF** (alias)
 - **diagrams.aws.network.DirectConnect**
 - **diagrams.aws.network.ElasticLoadBalancing**, **ELB** (alias)
+- **diagrams.aws.network.ElbApplicationLoadBalancer**
+- **diagrams.aws.network.ElbClassicLoadBalancer**
+- **diagrams.aws.network.ElbNetworkLoadBalancer**
```

Comment:
Perhaps removing the "Elb" prefix might be more user-friendly. 🤔

Reply:
How about alias these nodes?